app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,76 @@
+<p>Appwrite uses emails messages to perform user invite, user verification, login, and reset password. Emails messages can be customized to fit your app's design and voice.</p>
+
+<h2><a href="#customize" id="customize">Customize Templates</a></h2>
+<p>You can customize email templates in the Appwrite console.</p>
+<div class="notice">
+    <h2>Custom SMTP Server Required</h2>
+    <p>The built-in email service does not support custom email templates. Configure a <a href="#smtp">custom SMTP server</a> to enable custom email templates.</p>
+</div>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In your project, navigate to the <b>Auth</b> service.</li>
+    <li>Under the <b>Auth</b> service, navigate to the <b>Templates</b> tab.</li>
+    <li>Expand the email template you want to edit.</li>
+    <li>Select the <b>Template language</b> [TODO: @damodar is this used for localization? Can you configure a different template for each language?].</li>
+    <li>Update the email template fields and click **Update** to save your changes.</li>
+</ol>
+<h2><a href="#email-templates" id="email-templates">Email Templates</a></h2>
+<p>You can customize the email emplates for account verification, magic-url authentication, password resets, and user invites.</p>
+<h3><a href="#email-template-components" id="email-template-components">Email Template Components</a></h3>
+<p>Each email templates has the following components that you can customize.</p>
+<h2 id="permission-types"><a href="/docs/permissions#permission-types">Permission Types</a></h2>
+<p>In Client and Server SDKs, you will find a <b>Permission</b> class with helper methods for each role described below.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Component</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Sender name</td>
+        <td>[TODO: @damodar where is this used?].</td>
+    </tr>
+    <tr>
+        <td>Sender email</td>
+        <td>[TODO: @damodar which part of this can be configured? Does it have to do with the SMTP configuration?].</td>
```

Comment:
@lohanidamodar

Reply:
Whole email is configurable, but they should be careful, as the SMTP config should allow this email as the sender email. And whole custom email templates can only be changed when custom SMTP configuration is setup